utilities/oadp.py

```
@@ -137,3 +177,159 @@ def create_rhel_vm(
+                status=self.Status.COMPLETED,
+                timeout=self.timeout,
+            )
+        return self
+
+    def __exit__(self, exception_type, exception_value, traceback) -> None:
+        try:
+            if self.teardown:
+                try:
+                    delete_velero_resource(resource=self, client=self.client)
+                except Exception:
+                    pass
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)